integration/networktest/tests/helpful/l1_test.go

```diff
@@ -0,0 +1,44 @@
+package helpful
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/obscuronet/go-obscuro/integration/networktest"
+	"github.com/obscuronet/go-obscuro/integration/networktest/actions"
+	"github.com/obscuronet/go-obscuro/integration/networktest/env"
+)
+
+// super basic test that verifies it can connect to the L1 client and sees block numbers increasing (useful to sanity check testnet issues etc.)
+func TestL1IsAvailableAndProducingBlocks(t *testing.T) {
+	networktest.TestOnlyRunsInIDE(t)
+	networktest.Run(
+		"l1-availability",
+		t,
+		env.DevTestnet(),
+		actions.RunOnlyAction(func(ctx context.Context, network networktest.NetworkConnector) (context.Context, error) {
+			client, err := network.GetL1Client()
+			if err != nil {
+				return nil, err
+			}
+			blockStart, err := client.BlockNumber()
+			if err != nil {
+				return nil, err
+			}
+			fmt.Println("current block", blockStart)
+			time.Sleep(20 * time.Second)
```

Comment:
I think this is fine. 
Just wondering though. 
If we were to write this using the run/verify pattern, could it be done with a normal action, where run sets the blockStart in the context, then the test is set to run for 20seconds, and then the verify tests that what's in context (blockStart)  < blockEnd

Reply:
up to you. I'm happy with it like this. Does the job